include/swift/AST/Stmt.h

```diff
@@ -943,24 +986,47 @@ class CaseStmt final : public Stmt,
 
   llvm::PointerIntPair<Stmt *, 1, bool> BodyAndHasBoundDecls;
 
+  /// Set to true if we have a fallthrough.
+  ///
+  /// TODO: Once we have CaseBodyVarDecls, use the bit in BodyAndHasBoundDecls
```

Comment:
To be clear here: HasBoundDecls is true iff we have CaseBodyVarDecls to my commit adding those will fix that. I just did not want to create a field called ``BodyAndHasBoundDeclsAndHasFallthrough``...